k8-util/helm/fluvio-sys/templates/crd_managed_connector.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: managedconnectors.fluvio.infinyon.com
+spec:
+  group: fluvio.infinyon.com
+  scope: Namespaced
+  names:
+    kind: ManagedConnector
+    plural: managedconnectors
+    singular: managedconnector
+  versions:
+    - name: v1
+      served: true
+      storage:  true
+      subresources:
+          status: {}
+      schema:
+        openAPIV3Schema:
+          required: ["spec"]
+          type: object
+          properties:
+            status:
+              type: object
+              x-kubernetes-preserve-unknown-fields: true
+            spec:
+              type: object
+              required: ["name", "type"]
+              properties:
+                name:
+                  type: string
+                  minimum: 2
+                  maximum: 100
+                type:
+                  type: string
+                  minimum: 2
+                  maximum: 100
+                topic:
+                  type: string
+                  minimum: 2
+                  maximum: 100
+                parameters:
+                  type: object
+                  x-kubernetes-preserve-unknown-fields: true
+                secrets:
+                  type: object
+                  x-kubernetes-preserve-unknown-fields: true
+              x-kubernetes-preserve-unknown-fields: true
```

Comment:
Do we need `x-kubernetes-preserve-unknown-fields` here? It would allow arbitrary fields in the spec, I would think that's not desirable.

Reply:
Ah okay. Then not.